setup.py

```diff
@@ -35,10 +34,13 @@
             'pytest-mock',
             'Sphinx',
             'sphinx_rtd_theme',
-
             'six>=1.10.0,<2.0',
+        ],
+        'tornado': [
+            'tornado<6',
```

Comment:
can this be based on python version, instead of reducing the opportunities with python3?